pkg/operatorsource/configuring.go

```diff
@@ -95,6 +95,8 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *v1alpha1.Oper
 	}
 
 	cscExisting.EnsureGVK()
+	cscExisting.RemoveOwner(in.GetUID())
```

Comment:
Please add a comment mentioning why you are removing the owner. Looks good to me otherwise.

Reply:
@aravindhp done